packages/ionic-angular/src/schematics/application/schema.json

```diff
@@ -21,7 +21,7 @@
     },
     "unitTestRunner": {
       "type": "string",
-      "enum": ["karma", "none"],
+      "enum": ["jest", "karma", "none"],
       "description": "Test runner to use for unit tests.",
       "default": "karma"
```

Comment:
Can you make Jest the default? That's the default in `@nrwl/angular` and I am guessing it will be a crowd favorite going forward.